tdrs-backend/tdpservice/users/test/test_api/test_login_redirect_oidc.py

```diff
@@ -0,0 +1,43 @@
+"""Test the LoginRedirectAMS class."""
+from unittest import mock
+from tdpservice.users.api.login_redirect_oidc import LoginRedirectAMS
+
+@mock.patch("requests.get")
+def test_get_ams_configuration(requests_get_mock):
+    """Test the LoginRedirectAMS class."""
+    requests_get_mock.return_value.status_code = 200
+    requests_get_mock.return_value.json.return_value = {"key": "test"}
+    returned_value, error = LoginRedirectAMS.get_ams_configuration()
```

Comment:
So while I see all references of `get_ams_configuration()` get some version of `variable, error = `, it seems that error variable is not used? Maybe I'm missing something?

Reply:
No you are not. My intention for this review was to get early review and pick your opinion on drifting from using nginx as error response handler.